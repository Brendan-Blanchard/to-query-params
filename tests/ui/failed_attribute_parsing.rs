use to_query_params::QueryParams;

#[derive(QueryParams)]
struct Data {
    #[query(###)]
    number: f64,
}

fn main() {}
