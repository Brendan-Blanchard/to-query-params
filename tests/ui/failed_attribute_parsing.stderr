error: proc-macro derive panicked
 --> tests/ui/failed_attribute_parsing.rs:3:10
  |
3 | #[derive(QueryParams)]
  |          ^^^^^^^^^^^
  |
  = help: message: Unsupported attribute found in #[query(...)] attribute: Error("unexpected token in nested attribute, expected ident")
